NFS2SE.asm

```diff
@@ -42015,8 +42015,7 @@ loc_421DE3:
 	mov cl, bl
```

Comment:
I think these 2 lines could be removed now:
```asm
mov cl, bl
mov edi, 1
```

I added offset in C code to simulate joystick with more buttons and It's working!

Reply:
I removed these 2, too. Let's hope it'll work without them :sweat_smile:

Reply:
True, good catch! I didn't bother deleting them in my standalone patch so I patch as little as possible, but here it makes perfect sense to get rid of them.